src/server/http/index.js

```diff
@@ -4,9 +4,6 @@ import _ from 'lodash';
 import fs from 'fs';
 import Boom from 'boom';
 import Hapi from 'hapi';
-import HapiTemplates from 'vision';
-import HapiStaticFiles from 'inert';
-import HapiProxy from 'h2o2';
```

Comment:
These imports are not used here; instead they're used in https://github.com/elastic/kibana/blob/master/src/server/http/register_hapi_plugins.js